tmux.conf

```diff
@@ -24,12 +24,11 @@ bind -r L resize-pane -R 5
 
 # Smart pane switching with awareness of vim splits.
 # Source: https://github.com/christoomey/vim-tmux-navigator
-is_vim='echo "#{pane_current_command}" | grep -iqE "(^|\/)g?(view|n?vim?x?)(diff)?$"'
-bind -n C-h run "(tmux display-message -p '#{pane_current_command}' | grep -iqE '(^|\/)g?(view|nvim?)(diff)?$' && tmux send-keys C-h) || tmux select-pane -L"
-bind -n C-j run "(tmux display-message -p '#{pane_current_command}' | grep -iqE '(^|\/)g?(view|nvim?)(diff)?$' && tmux send-keys C-j) || tmux select-pane -D"
-bind -n C-k run "(tmux display-message -p '#{pane_current_command}' | grep -iqE '(^|\/)g?(view|nvim?)(diff)?$' && tmux send-keys C-k) || tmux select-pane -U"
-bind -n C-l run "(tmux display-message -p '#{pane_current_command}' | grep -iqE '(^|\/)g?(view|nvim?)(diff)?$' && tmux send-keys C-l) || tmux select-pane -R"
-bind -n C-\ run "(tmux display-message -p '#{pane_current_command}' | grep -iqE '(^|\/)g?(view|nvim?)(diff)?$' && tmux send-keys 'C-\\') || tmux select-pane -l"
+bind -n C-h run "(tmux display-message -p '#{pane_current_command}' | grep -iqE '(^|\/)g?(view|nvim?x?)(diff)?$' && tmux send-keys C-h) || tmux select-pane -L"
```

Comment:
in linux you can install a variant of vim called vimx or vim-X11. I had to do it because the normal vim i have doesn't have system clipboard in it. It comes down to a compiler flag you pass which determines what parts of vim you have, and vimx has clipboard where normal vim8 doesn't